PusherServer/BatchEvent.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace PusherServer
+{
+    /// <summary>
+    /// Represents an event that is part of a batch trigger
+    /// </summary>
+    public class BatchEvent
+    {
+        /// <summary>
+        /// The name of the event
+        /// </summary>
+        public string name { get; set; }
+
+        /// <summary>
+        /// The event data
+        /// </summary>
+        public string data { get; set; }
```

Comment:
We can probably fix the property naming to match the convention by tuning the JSON serialiser (you'll see snake case property names elsewhere at the moment as well), but I think this is out of scope for this branch.